thrift/history.thrift

```diff
@@ -857,4 +857,40 @@ service HistoryService {
       6: ShardOwnershipLostError shardOwnershipLostError,
       7: shared.EntityNotExistsError entityNotExistError,
     )
+
+  /**
+  * ReadDLQMessages returns messages from DLQ
+  **/
+  replicator.ReadDLQMessagesResponse ReadDLQMessages(1: replicator.ReadDLQMessagesRequest request)
```

Comment:
Does it make sense to name these ReadReplicatorDLQMessages ? If there is any other type of DLQ in the future, then the naming becomes confusing.

Reply:
My plan is to create a generic API for DLQ. The request contains a type to support multiple types.

Reply:
Makes sense. In that case is the return type 'replicator.ReadDLQMessagesResponse' also generic ?

Reply:
Yes. It also includes the DLQ type